packages/admin/admin/src/form/FinalFormCurrencyInput.tsx

```diff
@@ -0,0 +1,87 @@
+import { InputBase, InputBaseProps } from "@mui/material";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+import { useIntl } from "react-intl";
+
+import { ClearInputAdornment } from "../common/ClearInputAdornment";
+
+export type FinalFormCurrencyInputProps = InputBaseProps &
+    FieldRenderProps<number> & {
+        clearable?: boolean;
+    };
+
+export function FinalFormCurrencyInput({
```

Comment:
Should the currency symbol always be defined in the application, as a start-/end-adornment?
If so, maybe this component should be named NumberField instead 🤔

Reply:
Yes, good point. I added an optional prop for the decimals, so that it is possible to set a specific number of decimals (e.g. for the currency input, I only want to decimals to be displayed)